.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,94 @@
+# SPDX-FileCopyrightText: Copyright (c) 2022 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Bug Report
+description: File a bug report for cuSpatial
+title: "[BUG]: "
+labels: ["bug", "? - Needs Triage"]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to help cuSpatial and fill out this bug report!
+
+  - type: input
+    id: version
+    attributes:
+      label: Version
+      description: What version of cuSpatial are you running?
+      placeholder: "example: 22.10"
+    validations:
+      required: true
+
+  - type: dropdown
+    id: installation-method
+    attributes:
+      label: On which installation method(s) does this occur?
+      multiple: true
+      options:
+        - Docker
+        - Conda
+        - Source
+
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the issue
+      description: Please provide a complete and succinct description of the problem, including what you expected to happen.
+      placeholder: XYZ occured, I expected QRS results
+    validations:
+      required: true
+
+  - type: textarea
+    id: mvr
+    attributes:
+      label: Minimum reproducible example
+      description: Please supply a [minimum reproducible code example](https://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports) here.
+      render: shell
+
+  - type: textarea
+    id: logs
+    attributes:
+      label: Relevant log output
+      description: Please paste relevant error and log output here
+      render: shell
+
+  - type: textarea
+    id: env-details
+    attributes:
+      label: Environment details
+      description: Please provide any relevant environment details
+      placeholder: |
+       Environment location: [Bare-metal, Docker, Cloud(specify cloud provider)]
+       If method of install is Docker, provide `docker pull` & `docker run` commands used
```

Comment:
Cudf has a detailed environment report that is produced when one runs `$CUDF_HOME/print_env.sh` which we might want to piggy-back on for this. It prints an HTML output with hundreds of lines that is hidden in the issue report by default.

Reply:
Think we should we add print_env.sh to cuspatial, or refer users to cudf's?

Reply:
Will create an issue to make a print_env.sh, and then update the templates when merged.